content/en/docs/installing-on-linux/introduction/storage-configuration.md

```diff
@@ -1,37 +1,85 @@
 ---
 title: "Persistent Storage Configuration"
-keywords: 'Kubernetes, docker, KubeSphere, storage, volume, PVC'
+keywords: 'Kubernetes, docker, KubeSphere, storage, volume, PVC, KubeKey, add-on'
 description: 'Persistent Storage Configuration'
 
 linkTitle: "Persistent Storage Configuration"
 weight: 2140
 ---
+
 ## Overview
-Persistent volumes are a **Must** for KubeSphere. Therefore, before the installation of KubeSphere, a **default** 
-[StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) and corresponding storage plugin should be installed on the Kubernetes cluster. As different users may choose different storage plugins, you can use [KubeKey](https://github.com/kubesphere/kubekey) to install storage plugins through 
-[add-on](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md). This tutorial will introduce add-on configurations for some mainly used storage plugins.
+Persistent volume is **Must** for KubeSphere. [KubeKey](https://github.com/kubesphere/kubekey) let KubeSphere installed on different storage system by the 
+[add-on mechanism](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md). General steps of KubeKey to installing KubeSphere on Linux are like:
+
+1. Install Kubernetes
+2. Install **add-on** plugin for KubeSphere
+3. Install Kubesphere by [ks-installer](https://github.com/kubesphere/ks-installer)
```

Comment:
No mention of KubeKey in these 3 steps. Why we need it?

Reply:
KubeKey use ks-installer to install KubeSphere